apps/web/app/(resend_webhooks)/api/webhooks/bounce/route.ts

```diff
@@ -0,0 +1,58 @@
+import { getSupabaseClient } from 'db/src/supabase/getSupabaseClient'
+import { Webhook } from 'svix'
+
+const getWebhook = () => {
+  // eslint-disable-next-line turbo/no-undeclared-env-vars
+  const webhookSecret = process.env.RESEND_WEBHOOK_SECRET
+  if (!webhookSecret) throw new Error('RESEND_WEBHOOK_SECRET is not settled')
+  return new Webhook(webhookSecret)
+}
+
+type EmailType =
+  | 'email.sent'
+  | 'email.delivered'
+  | 'email.delivery_delayed'
+  | 'email.complained'
+  | 'email.bounced'
+  | 'email.opened'
+  | 'email.clicked'
+interface WebhookEvent {
+  created_at: string
+  data: {
+    created_at: string
+    email_id: string
+    from: string
+    subject: string
+    to: string[]
+  }
+  type: EmailType
+}
+
+export const POST = async (req: Request) => {
+  const webhook = getWebhook()
+  const svix_id = req.headers.get('svix-id') ?? ''
+  const svix_timestamp = req.headers.get('svix-timestamp') ?? ''
+  const svix_signature = req.headers.get('svix-signature') ?? ''
+
+  const body = await req.text()
+  const payload = webhook.verify(body, {
+    'svix-id': svix_id,
+    'svix-timestamp': svix_timestamp,
+    'svix-signature': svix_signature,
+  }) as WebhookEvent
```

Comment:
What happens if this verification fails?

We don't have a try catch here... We should return 500

Reply:
The SVIX validates the header and throws an error if it is not valid and sends a status 500.

I created an example without necessary headers and the svix thow this error:
![image](https://github.com/ocodista/trampar-de-casa/assets/68869379/f459b5f2-9085-438a-b2d7-26797496ad36)